src/api/shared/http-utils.ts

```diff
@@ -0,0 +1,6 @@
+export {
+  sharedFetch,
+  setFetchOptions,
+  setQueryParams,
```

Comment:
I would not put this in the API, even though it's useful it's also specific to what we do inside the library and we need toe flexibility to change its behavior without notice